tools/run_tests/jobset.py

```diff
@@ -229,15 +226,12 @@ def state(self, update_cache):
       self._tempfile.seek(0)
       stdout = self._tempfile.read()
       filtered_stdout = _filter_stdout(stdout)
```

Comment:
I'm afraid without truncating, the resultset object may be unnecessarily too big to hold in memory if you run all the tests many times.